packages/@webex/internal-plugin-ediscovery/src/transforms.js

```diff
@@ -361,6 +361,23 @@ class Transforms {
           }
         }
 
+        // Decrypt encrypted text map if present
+        if (activity.encryptedTextKeyValues !== undefined) {
+          activity.encryptedTextKeyValues.forEach((encryptedData, key) => {
```

Comment:
forEach will create a function for every key and value(encryptedData). After the decryption promise has returned we write back to the map by key with the decrypted text.

Reply:
Is this meant to be the other way around? i.e.

    (key, encryptedData) =>

Reply:
![image](https://user-images.githubusercontent.com/5473486/203808438-d967d0a5-1ddb-48c9-869f-127b8d1fd82c.png)

This is new, I'm not sure why the web-client is involved here. But we had to get reviews from the webex-js-sdk team in the past. So this is not too surprising. i.e. My review alone will not be enough.

Reply:
No, it's correct, but it does look wrong. Intelij helps here
![image](https://user-images.githubusercontent.com/44360726/203809945-7a2de10a-cbae-4055-8a8f-022790964d7f.png)